source/op/descrpt_se_a_mask.cc

```diff
@@ -0,0 +1,337 @@
+
+#include "custom_op.h"
+#include "ComputeDescriptor.h"
+#include "neighbor_list.h"
+#include "fmt_nlist.h"
+#include "errors.h"
+#include <cmath>
+
+typedef double boxtensor_t;
+typedef double compute_t;
+
+REGISTER_OP("DescrptSeAMask")
+    .Attr("T: {float, double} = DT_DOUBLE")
+    .Input("coord: T")
+    .Input("type: int32")
+    .Input("mask: int32")
+    .Attr("total_atom_num: int")
```

Comment:
What's the difference between this attr and the length of `type`?

Reply:
In principle, there is no different between the length of `type` and `total_atom_num`. The attr here is used for C API inference in `DeepPotMaskModel` and input data dimension checking.